packages/augur-ui/src/modules/app/actions/sync-blockchain.ts

```diff
@@ -40,6 +40,7 @@ export const syncBlockchain = (cb: Function) => (
 
   cb && cb();
 
+  // @ts-ignore
   augur.augurNode.getSyncData((err: any, res: any) => {
```

Comment:
There's an Augur.getSyncData on master. Is that OK to use?

Reply:
We’ll have a specific effort to pull out the augur-node stuff.